examples/flutter_gallery/pubspec.yaml

```diff
@@ -12,9 +12,9 @@ dependencies:
   intl: 0.15.7
   connectivity: 0.3.2
   string_scanner: 1.0.4
-  url_launcher: 3.0.3
+  url_launcher: 4.0.1
```

Comment:
Do you know why our upgrade script keeps flipping `url_launcher` back'n'forth between `3.x` and `4.x`?

Reply:
I'm not aware of a good reason for this, though rolling to a higher version seems right :)